Code/Common/src/sitkImage.cxx

```diff
@@ -139,6 +140,64 @@ namespace itk
       return this->m_PimpleImage->GetNumberOfPixels();
     }
 
+  unsigned int Image::GetSizeOfPixelComponent( ) const
+    {
+    switch( this->GetPixelIDValue() )
+      {
+      case sitkUnknown:
```

Comment:
Are both the sitkUnknown and default portions of the switch required? Possibly remove the `case sitkUnknown`?

Reply:
done